AudioKit/Common/Playgrounds/Synthesis.playground/contents.xcplayground

```diff
@@ -1,36 +1,2 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<playground version='6.0' target-platform='ios' display-mode='rendered'>
-    <pages>
-        <page name='Table Of Contents'/>
-        <page name='Amplitude Envelope'/>
-        <page name='Clarinet'/>
-        <page name='Drawbar Organ'/>
-        <page name='Dripping Sounds'/>
-        <page name='Drum Synthesizers'/>
-        <page name='Filter Envelope'/>
-        <page name='Flute'/>
-        <page name='FM Oscillator'/>
-        <page name='FM Oscillator Bank'/>
-        <page name='FM Oscillator Operation'/>
-        <page name='Mandolin'/>
-        <page name='Morphing Oscillator'/>
-        <page name='Morphing Oscillator Bank'/>
-        <page name='Noise Generators'/>
-        <page name='Noise Operations'/>
-        <page name='Oscillator'/>
-        <page name='Oscillator Synth'/>
-        <page name='Oscillator Bank'/>
-        <page name='Pedestrians'/>
-        <page name='Phase Distortion Oscillator'/>
-        <page name='Phase Distortion Oscillator Bank'/>
-        <page name='Phasor Operation'/>
-        <page name='Plucked String'/>
-        <page name='Plucked String Operation'/>
-        <page name='PWM Oscillator'/>
-        <page name='PWM Oscillator Bank'/>
-        <page name='Sawtooth Wave Oscillator Operation'/>
-        <page name='Segment Operations'/>
-        <page name='Sporth Based Generator'/>
-        <page name='Telephone'/>
-    </pages>
-</playground>
\ No newline at end of file
+<playground version='6.0' target-platform='ios' display-mode='rendered'/>
```

Comment:
Can you revert this change?  Good to have the pages in there and it looks like your Xcode just accidentally bashed this.